### PR TITLE
[ENH] Add transmit and receive RF field maps

### PR DESCRIPTION
**Pull request by agahkarakuzu**
B1<sup>+</sup> and B1<sup>-</sup> fields map are commonly used to correct for quantification errors stemming from transmit (excitation profile) and receive field (sensitivity profile) inhomogeneities.  

As in the qMRI methods specified under the `Anatomy Imaging Data` section, applications to generate RF field maps also call for the use of `grouping suffixes`. Nonetheless, this type of data does not convey considerable structural information (on purpose). Therefore, they are placed at the `fmap` folder. 

* Update descriptions of `B1plusmap` and `B1minusmap` in the `anatomy imaging data section` and give hyperlink to the `Appendix` for detailed informarion re fieldmaps. 
* Additions to the qMRI appendix
     * Add explanation about the purpose of the application
     * Add grouping suffix table for RF field map inputs 
    *  Define entity mappings for the grouping suffixes 
    *  Add explanation about where to place derived RF field maps.

### CODE REVIEW

File: src/99-appendices/10-qmri.md [hunk 1]
```diff
@@ -225,4 +225,170 @@ sub-01_T1map.json
...
+To properly identify constituents of this particular method, values of the `echo`
+entity MUST index the images as follows:  
+
+|`echo-1`|`echo-2`|
+|:--|:----|
+|Lower `EchoTime` | Higher `EchoTime`                  |
+|Spin Echo (SE) image|Stimulated Echo (STE) image|
+
+
+```
```
Comment (agahkarakuzu) [inline src/99-appendices/10-qmri.md:288]: Fix this

File: src/99-appendices/10-qmri.md [hunk 2]
```diff
@@ -225,4 +225,170 @@ sub-01_T1map.json
...
+
+|`run-1`|`run-2`|
+|:--|:----|
+|Body coil | Head coil|
+
+
+```
+└── sub-01/
+     └── fmap/
+         ├── sub-01_run-1_RB1COR.nii.gz (Body coil)
```
Comment (agahkarakuzu) [inline src/99-appendices/10-qmri.md:355]: Change this one with `acq`

**Comment by agahkarakuzu** [comment]
@ChristophePhillips @lazaral 

I made the changes for the use of `acq` for the particular use case of `RB1COR`:

|Body coil|Head coil|Use case|
|:--|:----| :----| 
|`_acq-body`|`_acq-head`|Single acquisition|
|`_acq-bodyMTw`|`_acq-headMTw`|`MTw` for `MPM`|
|`_acq-bodyPDw`|`_acq-headPDw`|`PDw` for `MPM`|
|`_acq-bodyT1w`|`_acq-headT1w`|`T1w` for `MPM`|

I used `bodyMTw` instead of `MTwBody` to have `body` and `head` as the leading word in the value, so that the processing logic becomes more inclusive to non-MPM applications. 

If this is reasonable, @Gilles86 I reflected on few other minor details we discussed in the meeting, should be ready to merge.

**Comment by lazaral** [comment]
Looks good to me!

**Comment by ChristophePhillips** [comment]
I can live with that. 😄

**Pull request merged**